ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/suites/CommandDeduplicationIT.scala

```diff
@@ -248,6 +243,7 @@ final class CommandDeduplicationIT(
       ledger: ParticipantTestContext,
       party: Party,
       delay: DelayMechanism,
+      skews: FiniteDuration,
```

Comment:
Why is it plural?

Reply:
Because it basically adds min and max skew

Reply:
How about `skewSum`, `totalSkew`, `minPlusMaxSkew` or `[total]skewUpperBound`?